sites/public/src/pages/account/edit.tsx

```diff
@@ -162,8 +161,6 @@ const Edit = () => {
           headingPriority={1}
         >
           <>
-            <SiteAlert type="notice" dismissable />
```

Comment:
Curious why this wasn't replaced?

Reply:
@emilyjablonski So these were essentially containers to load in the old "toast" (aka AlertBox sticky) messages and display them if necessary. Since we're handling that now at the layout level, we don't need an equivalent to SiteAlert on individual pages.